src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/21.05/user-account-project-subscription.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
Just need to move this file under 22.05 changesets instead of 21.05

Reply:
Please see d26411b2f550895fb32bee12bd82f6339e648e4b & 681cc8d3fb768bf618e71609f861895b486fd40b.